ui/components/multichain/multichain-connected-site-menu/multichain-connected-site-menu.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useSelector } from 'react-redux';
+import {
+  BackgroundColor,
+  BorderColor,
+  BorderRadius,
+  IconColor,
+  Size,
+} from '../../../helpers/constants/design-system';
+import { BadgeWrapper, Icon, ICON_NAMES } from '../../component-library';
+import Box from '../../ui/box';
+import { getSelectedIdentity } from '../../../selectors';
+import Tooltip from '../../ui/tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export const MultichainConnectedSiteMenu = ({
+  className,
+  globalMenuColor,
+  status,
+  text,
+}) => {
+  const t = useI18nContext();
+  const selectedAccount = useSelector(getSelectedIdentity);
+  return (
+    <Box
+      className={classNames('multichain-connected-site-menu', className)}
+      data-testid="connection-menu"
+    >
+      <Tooltip
+        title={
+          status === 'STATUS_NOT_CONNECTED'
+            ? t('statusNotConnectedAccount')
+            : `${selectedAccount?.name} ${text?.toLowerCase()}`
```

Comment:
Rather than manipulate strings like this, I think it's better to maybe do something like: 

`${selectedAccount?.name} - ${text}`

Reply:
Hmm! I agree but the reason I manipulated the string here is because in case of `connected `and `connected to another account` state I had to show the tooltip text as `Account 3 connected` and `Account 3 not connected` respectively, as in only the first letter is uppercase. We already had a key defined in json files as `statusConnected` and `statusNotConnected` and I didn't want to add another key since we can update these existing ones when we will be removing the existing indicator. But I am open to updating these. Wdyt?

Reply:
I think we should either create new strings or use old ones -- modifying localized strings with `toUpperCase`, etc. seems bad, and may not apply correctly in all languages.

Reply:
Aah, my bad! I didn't think of other languages. Thanks a lot  David. Updated here [90affbd](https://github.com/MetaMask/metamask-extension/pull/18167/commits/90affbd1e63dec216a1c7edd2445c6839c584794)